crates/proto/src/event.rs

```diff
@@ -43,9 +46,9 @@ pub trait ProtoEvent: Message + Name + Serialize + DeserializeOwned + Sized {
         // NOTE: Is there any condition where there would be duplicate EventAttributes and problems that fall out of that?
         let mut attributes = HashMap::<String, serde_json::Value>::new();
         for attr in &event.attributes {
-            let value = serde_json::from_str(&attr.value)
+            let value = serde_json::from_slice(attr.value_bytes())
                 .with_context(|| format!("could not parse JSON for attribute {:?}", attr))?;
-            attributes.insert(attr.key.clone(), value);
+            attributes.insert(String::from_utf8_lossy(attr.key_bytes()).into(), value);
```

Comment:
This seemed to be the least invasive way to deal with tendermints new flexibility around utf8. In the forst case there will be some weird symbols in the attributes. But that's probably fine?